pkg/rke/k8s_version_info.go

```diff
@@ -656,6 +656,10 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.7.5-patch0",
 			MinRKEVersion:     "1.4.0-rc0",
 		},
+		"v1.27": {
+			MinRancherVersion: "2.7.5-patch0",
```

Comment:
Need confirmation for whether k8s 1.27 should be available in Rancher v2.7.5 or v2.7.6

Reply:
k8s 1.27 issue is for 2.8 milestone, so this would be v2.8.0 and RKE v1.5.x...

Reply:
Alright! Thanks. Will update the PR